source/run-agg-pipelines.txt

```diff
@@ -140,69 +181,45 @@ First Stage
      The ``$project`` pipeline in the example suppresses the ``_id``
      field from the documents it passes to the next stage.
 
+  The ``$project`` pipeline in the example transforms the example
+  document and passes the following output to the next pipeline stage:
+
+  .. code-block:: javascript
+     :linenos:
+     :copyable: false
+
+     {
+       imdb: { rating: 8.2, votes: 419589, id: 83658 },
+       genres: [ 'Sci-Fi', 'Thriller' ],
+       title: 'Blade Runner'
+     }
+
 Second Stage
   The :pipeline:`$unwind <$unwind>` stage passes a document for each
   element in the ``genres`` array to the next pipeline stage.
 
-  Consider the following document:
+  The ``$unwind`` pipeline in the example passes two transformations of
```

Comment:
I think "two transformations" is a little confusing and makes it sound like the `$unwind` stage is altering the documents in two distinct ways. Maybe say:

"The `$unwind` stage generates the following two documents from the original example document: "

Reply:
updated